### DISCUSSION TITLE
Where does Cmake tools persist default build

### DISCUSSION THREAD

**Discussion by musmuris** [microsoft/vscode-cmake-tools#3416] [General]
At https://github.com/microsoft/vscode-cmake-tools/blob/main/docs/build.md#build-the-default-target it says "CMake Tools persists a "default target" for the build process."

Where does that get persisted?  Is it under the configured build folder?

**Comment by gcampbell-msft** [comment]
It does not persist this anywhere in a file, instead it uses the VS Code concept of a `Memento` and global state. https://vscode-api.js.org/interfaces/vscode.Memento.html
